src/App.tsx

```diff
@@ -1,61 +1,118 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const todosWithUser: Todo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId) || null,
+  };
+});
+
+export const App: React.FC = () => {
+  const [titleInput, setTitleInput] = useState('');
+  const [userSelect, setUserSelect] = useState('');
+  const [isFormChecked, setFormChecked] = useState(false);
+  const [todos, setTodos] = useState(todosWithUser);
+
+  const MaxId = Math.max(...todos.map(todo => todo.id));
+
+  const resetForm = () => {
+    setTitleInput('');
+    setUserSelect('');
+    setFormChecked(false);
+  };
+
+  const addNewTodo = () => {
+    const selectedUser = usersFromServer
+      .find(user => user.name === userSelect) as User;
```

Comment:
What will happen if the method doesn't find anything? You can get undefined here. Add an `if` statement for the case when there is no selectedUser.
```suggestion
    const selectedUser = usersFromServer
      .find(user => user.name === userSelect);
      
    if (!selectedUser) ...
```